src/app/extensions/quoting/facades/quoting.facade.ts

```diff
@@ -28,7 +29,7 @@ export class QuotingFacade {
   }
   loading$ = this.store.pipe(select(getQuotingLoading));
 
-  quotingEntities$(options?: { automaticRefresh: boolean }) {
+  quotingEntities$(options: QuoteEntitiesOptions = { automaticRefresh: true }) {
```

Comment:
I would not use an options object here if it contains only one attribute. 

quotingEntities$( automaticRefresh: true ) ...

...would be fine.

Reply:
That was my first solution. I thought that the quotingEntities$ method with an object as optional parameter would be the better solution. The expression of ```this.facade.quotingEntities$({ automaticRefresh: false })``` is more explicit than ```this.facade.quotingEntities$(false)``` and understandable, what the method should do.